tests/tests.go

```diff
@@ -188,46 +187,31 @@ func setRange(rangeStart, rangeEnd string) error {
 	return nil
 }
 
-func DeleteLeaderManager() {
+func DeleteOneManager() {
```

Comment:
not sure about this, but maybe we should add to test a part that verifies that even though we deleted one manager, we continue to serve, keeps the functionality or something like that.
for example, in the previous revision, when there was a leader, after the deletion, there is a verification that there is still a healthy pod in the manager namespace (https://github.com/k8snetworkplumbingwg/kubemacpool/pull/109/commits/7e983b7906ba77a768f0618d3166760391440a5a#diff-4de094b4f2274c29e9f11102da4ca7d0L217-L230)
while here we just delete kubemacpool pod and...that's it. 
without it I'm not sure this test is needed, it is a bit "meat-less"